src/evo/cbtx.h

```diff
@@ -25,15 +25,21 @@ class CChainLocksHandler;
 // Forward declaration from core_io to get rid of circular dependency
 UniValue ValueFromAmount(const CAmount& amount);
 
+enum class CbTxVersion : uint16_t {
+    INVALID = 0,
+    MERKLE_ROOT_MNLIST = 1,
+    MERKLE_ROOT_QUORUMS = 2,
+    BEST_CHAINLOCK = 3,
```

Comment:
```suggestion
    BEST_CLSIG_AND_CREDIT_POOL = 3,
```
or something like that?

Reply:
hmm.. `BEST_CLSIG_AND_CP_BALANCE` maybe?

Reply:
I vote for `CLSIG_AND_CP_BALANCE`

Reply:
applied

Reply:
I'd vote to avoid any abbreviations of `CP` in the codebase 🙈

Reply:
oh god :D `CLSIG_AND_BALANCE`? any other suggestions?

Reply:
Probably fine